backend/src/main/kotlin/api/GetStoresHandler.kt

```diff
@@ -0,0 +1,29 @@
+package api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import database.Repository
+import io.javalin.http.Context
+import io.javalin.http.Handler
+
+class GetStoresHandler(private val repo: Repository) : Handler {
+
+    private val objectMapper = ObjectMapper()
+
+    init {
+        objectMapper.registerModule(JavaTimeModule());
+        objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    }
+
+    override fun handle(ctx: Context) {
+        val stores = repo.getStores()
+        val storesSeasons = repo.getStoreSeasons()
+        stores.map { store ->
+            store.seasons = storesSeasons
+                .filter { it.first == store.id }
+                .map { it.second }
+        }
+        ctx.result(objectMapper.writeValueAsString(stores)).contentType("application/json")
```

Comment:
all the stores seem to return an empty array for the `seasons` property: 

![image](https://user-images.githubusercontent.com/7469472/109699289-4ba7f800-7b88-11eb-9a6b-1a4f9edb6314.png)

Reply:
After some shallow investigation I think this is a bug that appears the first time the import is triggered.
Can you try to manually trigger the import (you can do so by hitting `localhost:7000/import`) and tell me if the problem persists? This process takes a while but hopefully it will return a `201` status code to the browser

I've also added a commit that prints how many stores, seasons & stores/seasons were added/updated, to help having a more clear idea on whats happening behind the curtains.
![image](https://user-images.githubusercontent.com/47791391/110032384-fb6b9a00-7d2f-11eb-9c69-1cf606d577c8.png)


If you want me to deep investigate this and trying to fix it just tell me :)